keras/layers/reshaping/reshape.py

```diff
@@ -53,8 +53,14 @@ def compute_output_spec(self, inputs):
             shape=output_shape, dtype=inputs.dtype, sparse=inputs.sparse
         )
 
+    def build(self, input_shape):
+        self.reshape_shape = operation_utils.compute_reshape_output_shape(
```

Comment:
We should avoid having this as a public attribute on the class. Please rename else, perhaps with a more explicit name like`_sample_output_shape` (it's the per-sample shape instead of the batch shape)

Reply:
Thanks! Done.